tests/conftest.py

```diff
@@ -6,6 +6,9 @@
 from eth_account import Account
 from eth_account.messages import encode_defunct
 
+# 235 premints starting from ID 0
+ID_AFTER_PREMINT = 235
```

Comment:
minor: wondering if it's cleaner to use the contract's `MAX_PREMINT` constant instead of repeating the definition here (but it would require `Llama` as an arg to the fixtures/tests)